### PR TITLE
Shell script to automate the databricks cluster deploy

### PR DESCRIPTION
**Pull request by Rajesh2015**
- Created the shell script which will create the docker container  read a configuration.yaml file and deploy the cluster with those configuration

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ORGANIZATION=rajeshlaerdal
```
Comment (henrikbulldog) [inline deploy.sh]: ORGANIZATION should be a parameter

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+REPOSITORY_NAME=anonymizertest
```
Comment (henrikbulldog) [inline deploy.sh]: REPOSITORY_NAME should be a parameter

File: databricks_deploy.sh [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+if [[ ${answer} = y ]]; then
+read -p "Enter the image name [${imagename}] : $name" imagename
+docker build  --build-arg SCALA_VERSION="scala-${SCALA_VERSION}" --no-cache -t imagename:local .
+fi
+
+
+deploy:
+#Running deploy script
+echo "Creating tag and pushing  the container to dockerhub"
+bash  $(pwd)/deploy.sh
```
Comment (henrikbulldog) [inline databricks_deploy.sh:75]: Should specify ORGANIZATION and REPOSITORY_NAME as parameters

**Review by henrikbulldog — CHANGES_REQUESTED** [review]

File: deploy.sh (hunk 2 above)
Comment (Rajesh2015) [reply deploy.sh]: Moved these to yaml file for easy configuration

**Review by henrikbulldog — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -7,11 +7,63 @@ Windows command scripts to deploy a Scala project to a Databricks cluster.
...
+deleteJarsTask := {
+  val folder = new File("lib/jars")
+  IO.delete(folder)
+
+}
+```
+- All your settings can be put inside a config.yaml file like below(Applicable only for shell script).
+```
+ORGANIZATION : <orgnaziation docker hub name>
+REPOSITORY_NAME : <repositort name>
```
Comment (henrikbulldog) [inline README.md:41]: Should be name of folder, i.e. analytics-spark-org-anonymize
Should be possible to use one config file for multiple projects

File: README.md [hunk 5]
```diff
@@ -7,11 +7,63 @@ Windows command scripts to deploy a Scala project to a Databricks cluster.
...
+VERSION : <version of repository>
+clustername: "<enter cluster name>"
```
Comment (henrikbulldog) [inline README.md:43]: Should be <user name>/REPOSITORY_NAME:VERSION

File: README.md [hunk 6]
```diff
@@ -7,11 +7,63 @@ Windows command scripts to deploy a Scala project to a Databricks cluster.
...
+ assume_role_arn: "<enter role>"
+}
+zone_id: "<enter region>"
+min_workers: "<min workers>"
+max_workers: "<max workers>"
+spark_version: "<spark version in cluster>"
+node_type_id : "<node type>"
+autotermination_minutes: "<time out>"
+driver_node_type_id : "<driver node type>"
+docker_image_url : "<docker image path>"
```
Comment (henrikbulldog) [inline README.md:56]: Should be <user name>/REPOSITORY_NAME:VERSION

File: README.md [hunk 7]
```diff
@@ -7,11 +7,63 @@ Windows command scripts to deploy a Scala project to a Databricks cluster.
...
+clustername: "<enter cluster name>"
+databricks_profile: "<enter profile>"
+roles: {
+ instance_profile_arn: "<enter role>"
+ assume_role_arn: "<enter role>"
+}
+zone_id: "<enter region>"
+min_workers: "<min workers>"
+max_workers: "<max workers>"
+spark_version: "<spark version in cluster>"
```
Comment (henrikbulldog) [inline README.md:52]: Can that be read from build.sbt?

**Pull request merged**